php/tests/ExampleCharacterTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Emilybache\StrangecharacterTestdesignKata;
+
+use Emilybache\StrangecharacterTestdesignKata\Character;
+use Emilybache\StrangecharacterTestdesignKata\CharacterFinder;
+use PHPUnit\Framework\TestCase;
+
+class ExampleCharacterTest extends TestCase
+{
+    public function testFindCharacterByLastName(): void
+    {
+        // This test constructs all its own test data to make it clearer what it does
+        $karen = new Character("Karen", "Wheeler");
+        $mike = new Character("Mike", "Wheeler");
+
+        // this is example code showing kinds of assertion you could do on a Character
+        $nancy = new Character("Nancy", "Wheeler");
+        $this->assertSame("Nancy", $nancy->firstName);
```

Comment:
Is it possible to declare some kind of equals method on the Character class so you don't need to do the call to firstName?

Reply:
Sure, I'll do that

Reply:
I've added the missing equality tests and use `equal` internally (in the `addChild` method).